client/src/components/SingleList.jsx

```diff
@@ -0,0 +1,57 @@
+import React from "react";
+import { useState, useEffect } from "react";
+import { MapContainer, TileLayer, Marker, useMap } from "react-leaflet";
+
+export default function SingleList(props) {
+  const [restName, setRestName] = useState({});
+  const [location, setLocation] = useState([0, 0]);
+  const [Lat, setLat] = useState(0);
+  const [long, setlong] = useState(0);
+  const [Check, setCheck] = useState(false);
```

Comment:
Just something I noticed, a lot of files have unused stuff in them. It is a good idea to go back and clean up an loose ends or old ideas periodically. 
For instance, in this block here all of line 7, line 10, useEffect on line 2, and useMap on line 3 are not being used in this component (signaled by being faded out in vs code).